dotCMS/src/main/java/com/dotcms/browser/BrowserAPI.java

```diff
@@ -223,5 +225,23 @@
      * @throws DotDataException
      */
     public Map<String, Object> getFolderContent(final BrowserQuery browserQuery) throws DotSecurityException, DotDataException;
-}
 
+
+	/**
+	 * Returns a collection of contentlets that live inside the parent(browserQuery.directParent)
+	 * @param browserQuery {@link BrowserQuery}
+	 * @return list of contentlets
+	 */
+	List<Contentlet> getContentUnderParentDB(BrowserQuery browserQuery);
```

Comment:
I would rename to `getContentUnderParentFromDB`

Reply:
Why not use as paraneter directly the directParent attribute?